src/applications/vaos/containers/TypeOfCarePage.jsx

```diff
@@ -90,6 +62,7 @@ export class TypeOfCarePage extends React.Component {
 
   hideAlert = () => {
     const showAlert = !this.state.showAlert;
+    this.props.onClickedUpdateAddressButton();
     this.setState({ showAlert });
```

Comment:
Now that this state is stored in Redux, you don't need this `showAlert` property. You should be able to remove any uses of it.

Reply:
Good point. Removing the need for showAlert property simplifies the code and logic. Thanks.